src/components/libraryoptionseditor/libraryoptionseditor.js

```diff
@@ -550,9 +550,9 @@ import template from './libraryoptionseditor.template.html';
     function getOrderedPlugins(plugins, configuredOrder) {
         plugins = plugins.slice(0);
         plugins.sort((a, b) => {
-            a = configuredOrder.indexOf(a.Name);
-            b = configuredOrder.indexOf(b.Name);
-            return a - b;
+          a = configuredOrder.indexOf(a.Name);
+          b = configuredOrder.indexOf(b.Name);
+          return a - b;
```

Comment:
```suggestion
            a = configuredOrder.indexOf(a.Name);
            b = configuredOrder.indexOf(b.Name);
            return a - b;
```

Reply:
The indentation checks are still disabled in a bunch of files.